.env

```diff
@@ -1 +1 @@
-REACT_APP_GOOGLE_CLIENT_ID=706375540729-sqnnig7o0d0uqmvav0h8nh8aft6l55u3.apps.googleusercontent.com
+VITE_GOOGLE_CLIENT_ID=706375540729-sqnnig7o0d0uqmvav0h8nh8aft6l55u3.apps.googleusercontent.com
```

Comment:
Out of curiosity, do you know if this is used in any extant code? IIRC, it had something to do with the Google Calendar integration that is broken pending #8.

Reply:
I think it can be taken out? Would we need a client ID to add back the gcal integration?

Reply:
yes, yes you would need it; though currently it's on my account